litmus-portal/backend/auth/pkg/models/user.go

```diff
@@ -141,6 +149,36 @@ func (uinfo *PublicUserInfo) GetEmail() string {
 	return uinfo.Email
 }
 
+// GetCreatedAt user createdAt
+func (uinfo *PublicUserInfo) GetCreatedAt() *time.Time {
```

Comment:
are these getter functions required? I can see the variables are exported so they can be directly accessed. Getter functions are used for private or inaccessible variables mostly.

Reply:
Having getter functions is a good practice in golang. You can see that in Kubernetes go API itself. ALl its variables are exported but still, we do have getter functions for them